kinesis-mock/src/main/scala/kinesis/mock/KinesisMockServiceConfig.scala

```diff
@@ -44,7 +44,7 @@ object KinesisMockServiceConfig {
       "password"
     )
     certPath <- env("KINESIS_MOCK_CERT_PATH").default(
-      "kinesis-mock/src/main/resources/server.json"
+      "server.json"
```

Comment:
Changed the default to the top-level as that is the configuration for the NPM package and the Docker image